packages/plugins/rum/src/built/rum-browser-sdk.ts

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+/* global globalThis */
+
+import { datadogRum } from '@datadog/browser-rum';
+
+// To please TypeScript.
+const globalAny: any = globalThis;
+
+// Also them to the global DD_RUM object.
+globalAny.DD_RUM = globalAny.DD_RUM || {};
+globalAny.DD_RUM = {
+    ...globalAny.DD_RUM,
```

Comment:
Now that I better understand the fact that we are using a pre-bundled version of the SDK, does it make sense to try to inject here both `...DD_RUM, ...datadogRum`? (As those 2 could be using different versions)

Why not just doing `globalAny.DD_RUM = datadogRum`? (also IIRC, even the npm datadogRum should write in DD_RUM 🤔)